frontend/templates/package.json.hbs

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "di-ipv-cri-{{ kebabCase criName }}-front",
+  "version": "1.0.0",
+  "description": "Frontend for the {{ titleCase criName }} Credential Issuer",
+  "main": "index.js",
+  "scripts": {
+    "start": "node src/app.js",
+    "start:ci": "API_BASE_URL=http://127.0.0.1:8050 REDIS_SESSION_URL= NODE_ENV=development node src/app.js",
+    "dev": "NODE_ENV=development nodemon src/app.js",
+    "lint": "eslint",
+    "build": "npm run build:sass && npm run build:js && npm run copy-assets",
+    "build:js": "npm run build:js:application && npm run build:js:cookies && npm run build:js:all",
+    "build:js:all": "mkdir -p dist/public/javascripts; uglifyjs node_modules/govuk-frontend/govuk/all.js node_modules/hmpo-components/all.js --beautify -o dist/public/javascripts/all.js",
+    "build:js:application": "mkdir -p dist/public/javascripts; uglifyjs src/assets/javascripts/application.js  node_modules/di-ipv-cri-common-express/src/assets/javascript/application.js --beautify -o dist/public/javascripts/application.js",
+    "build:js:cookies": "mkdir -p dist/public/javascripts; uglifyjs node_modules/di-ipv-cri-common-express/src/assets/javascript/cookies.js --beautify -o dist/public/javascripts/cookies.js",
+    "build:js:minify": "uglifyjs src/assets/javascript/application.js -o src/assets/javascript/application.js -c -m && uglifyjs src/assets/javascript/cookies.js -o src/assets/javascript/cookies.js -c -m",
+    "build:sass": "rm -rf dist/public/style.css && sass --no-source-map src/assets/scss/application.scss dist/public/stylesheets/application.css --style compressed",
+    "copy-assets": "mkdir -p dist && copyfiles -u 1 src/**/*.njk dist/ src/locales/**/** dist/ && copyfiles -u 3 src/assets/javascript/*.js dist/public/scripts",
+    "check-translation": "node node_modules/di-ipv-cri-common-express/scripts/checkTranslations.js",
+    "test": "jest"
+  },
+  "author": "",
+  "license": "MIT",
+  "devDependencies": {
+    "chai": "4.3.8",
+    "chai-as-promised": "7.1.1",
+    "copyfiles": "2.4.1",
+    "eslint": "8.49.0",
+    "eslint-config-prettier": "9.0.0",
+    "eslint-plugin-prettier": "5.0.0",
+    "jest": "29.7.0",
+    "nodemon": "3.0.1",
+    "npm-run-all": "4.1.5",
+    "prettier": "3.0.3",
+    "sass": "1.66.1",
+    "uglify-js": "3.17.4"
+  },
+  "dependencies": {
+    "aws-sdk": "2.1376.0",
+    "axios": "1.4.0",
+    "cfenv": "1.2.4",
+    "connect-dynamodb": "2.0.6",
+    "di-ipv-cri-common-express": "github:alphagov/di-ipv-cri-common-express#v1.0.0",
+    "dotenv": "16.3.1",
+    "express": "4.18.2",
+    "express-async-errors": "3.1.1",
+    "express-session": "1.17.3",
+    "govuk-frontend": "4.7.0",
+    "hmpo-app": "3.0.1",
+    "hmpo-components": "6.3.0",
+    "hmpo-form-wizard": "13.0.0",
+    "hmpo-i18n": "6.0.1",
+    "hmpo-logger": "7.0.1",
+    "nunjucks": "3.2.4"
```

Comment:
`prettier`, `cfenv` don't seem to be used by this PR

Reply:
- `prettier` has been removed as its covered by the `pre-commit` config
- `cfenv` is currently part of `common-express` and should be removed there first, and then in this file